scripts/FigS2B_adt-plots.R

```diff
@@ -0,0 +1,144 @@
+# This script is used to create the mini versions of the plots included in the ADT report
+# we use the results from SCPCS000216/SCPCL000290
+
+renv::load()
+
+library(ggplot2)
+library(SingleCellExperiment)
+
+theme_set(
+  theme_classic() +
+    theme(
+      strip.background = element_rect(fill = "transparent"),
```

Comment:
The strip borders look a little thick in the final image, this seems good when I run on my end.

```suggestion
      strip.background = element_rect(fill = "transparent", linewidth = 0.5 ),
```